.github/scripts/hermetic_library_generation.sh

```diff
@@ -92,6 +92,12 @@ pushd "${api_def_dir}"
 git checkout "${googleapis_commitish}"
 popd
 
+# generate changed libraries
```

Comment:
I think `generate changed libraries` is a little confusing, I thought we are actually generating libraries when I first read it, but we are actually comparing two configs and figuring out the libraries to generate.

Reply:
Changed to `get changed library list`